### ISSUE TITLE
Support for %S:%L time format.

### ISSUE THREAD

**Issue by stepusin** [fluent/fluent-bit#2367]
**Feature Request**
To add `[PARSER]` `Time_Format` support for `%S:%L` (in addition to the current `%S.%L`).

**Additional context**
Working on a project with a legacy software that logs in this format: 07/18 16:01:32:429

The theoretical `Time_Format` to support that would be: `%m/%d %H:%M:%S:%L` (which the documentation states is not supported).  This logs a warning, as expected of: "[ warn] [parser:parser-name] invalid time format %m/%d %H:%M:%S:%L for '07/18 16:01:32:429'".

I have tried the `Time_Format` `%m/%d %H:%M:%S.%L`, but it drops the fractional seconds to zero (1595088277.000000000)

I am not sure how common this format is, but it seems reasonable that other legacy code bases (this code base originated in the 90s) use a `:` after the seconds, although it is not a contemporary standard.  This is not something I cannot change at the logger configuration.

**Comment by gsmith-sas** [comment]
@stepusin Is this the same issue as #2210?  If so, it looks like work is underway.

**Comment by stepusin** [comment]
@gsmith-sas yes, this is the same request.  makes sense to allow any separator to accommodate, you can close this.

**Comment by gsmith-sas** [comment]
@fujimotos Based on @stepusin comment, it looks like you can close this issue as a duplicate of #2210.

**Comment by fujimotos** [comment]
@stepusin  @gsmith-sas Thank you for reporting. I close this ticket as
duplication of #2210.

In the meantime, I have implemented generic subsecond support in #2722,
so your issue should be resolved once that patch is merged into main.
